queue/queue.mbt

```diff
@@ -80,6 +108,25 @@ fn op_equal[T : Eq](self : Cell[T], other : Cell[T]) -> Bool {
   }
 }
 
+/// Tests if two queues are equal.
+fn op_equal[T : Eq](self : Queue[T], other : Queue[T]) -> Bool {
+  if self.length != other.length || self.first != other.first || self.last != other.last {
+    return false
+  }
```

Comment:
is this correct?

Reply:
I think it's correct.

ref op_equal for Cell[T]